convention.md

```diff
@@ -118,6 +118,12 @@ S: Indicates seconds, preceded by the number of seconds, if seconds are specifie
 - Contains a JSON string for transporting complex data formats that cannot be exposed as single value attributes.
 - The payload MUST be either a JSON-Array or JSON-Object type, for other types the standard Homie types should be used.
 
+#### Binary
+
+- Contains raw bytes of data.
+- Recommended for data that cannot be efficiently represented as string like pictures or audio samples.
+- More information about the data can be provided in the format.
+
```

Comment:
here we need to add that this doesn't have to be valid UTF-8. In other places where we say it has to be UTF-8, the exception should be added.